src/hotspot/cpu/aarch64/continuationFreezeThaw_aarch64.inline.hpp

```diff
@@ -129,6 +129,12 @@ void FreezeBase::adjust_interpreted_frame_unextended_sp(frame& f) {
   }
 }
 
+inline void FreezeBase::prepare_freeze_interpreted_top_frame(const frame& f) {
+  assert(*f.addr_at(frame::interpreter_frame_last_sp_offset) == 0, "should be null for top frame");
+  intptr_t* lspp = f.addr_at(frame::interpreter_frame_last_sp_offset);
+  *lspp = f.unextended_sp() - f.fp();
```

Comment:
```suggestion
  f.interpreter_frame_set_last_sp(f.unextended_sp());
```

Reply:
Changed, here and in the other platforms.